### PR TITLE
Graph: update value in legend when unit or decimals is changed in edi…

### LINKED ISSUE #17518: Current value in legend isn't updated when unit is changed in edit mode
**What happened**:

I was configuring a panel:
* Enabled table legend, right hand side, show current, it showed values like 0.025
* Changed unit to second

The current value did not change to reflect the new unit, only after I disabled and re-enabled show current.

**What you expected to happen**:
* Current values are updated in the preview to reflect the new unit

**Anything else we need to know?**:

**Environment**:
- Grafana version: 6.2.2
- Data source type & version: Prometheus 2.10.0
- OS Grafana is installed on: Ubuntu 18.04 (Docker)
- User OS & Browser: Arch Linux, Firefox 69.0a1

### PR DESCRIPTION
**Pull request by lzdw**
…t mode

**What this PR does / why we need it**:
 Enable legend in edit mode and then change unit or decimals , values in graph-legend should be updated. 
**Which issue(s) this PR fixes**:

Fixes #17518

**Special notes for your reviewer**:

### CODE REVIEW

File: public/app/plugins/panel/graph/graph.ts [hunk 1]
```diff
@@ -100,10 +100,11 @@ class GraphElement {
     const graphHeight = this.elem.height();
     updateLegendValues(this.data, this.panel, graphHeight);
 
+    if (this.legendElem.hasChildNodes()) {
+      ReactDOM.unmountComponentAtNode(this.legendElem);
```
Comment (torkelo) [inline public/app/plugins/panel/graph/graph.ts]: why unmount on every render? must be another way to fix this

Comment (lzdw) [reply public/app/plugins/panel/graph/graph.ts]: > why unmount on every render? must be another way to fix this

ok, i will try another solution.

Comment (torkelo) [reply public/app/plugins/panel/graph/graph.ts]: Do you know when you can do that?

Comment (lzdw) [reply public/app/plugins/panel/graph/graph.ts]: Yes, I've changed the solution, but I'm not sure that's a good solution.

**Pull request closed without merging**